### PR TITLE
Add org.gabmus.notorious

### PR DESCRIPTION
**Pull request by GabMus**

### CODE REVIEW

**Comment by GabMus** [comment]
bot, build org.gabmus.notorious

File: org.gabmus.notorious.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "command": "notorious",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home:rw"
```
Comment (barthalion) [inline org.gabmus.notorious.json:12]: What does it need `home:rw` for?

Comment (GabMus) [reply org.gabmus.notorious.json:12]: Users can choose a folder to use for notes in their home directory. This is useful if one wants to use a folder that's synced with an online service like Nextcloud. I personally use `~/Nextcloud/Documents/Notes`.

Besides the default is `~/Documents/Notes` and there's a good reason not to keep these notes inside of the sandbox like moving them around, sharing them or doing stuff like grep on them.

File: org.gabmus.notorious.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            	"fc-cache -fs"
+            ]
+        },
+        {
+            "name": "gnome-builder-gtksourceview4-colorschemes",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gnome-builder.git",
```
Comment (bilelmoussaoui) [inline org.gabmus.notorious.json:52]: Why not just use two sources of type file that points to the two color scheme files?

Comment (GabMus) [reply org.gabmus.notorious.json:52]: Guess I overthought it. Sure can do.

File: org.gabmus.notorious.json (hunk 1 above)
Comment (barthalion) [reply org.gabmus.notorious.json:12]: I noticed it in the settings after posting my quesiton. :+1:

File: org.gabmus.notorious.json (hunk 2 above)
Comment (GabMus) [reply org.gabmus.notorious.json]: just pushed it, let me rebuild

**Comment by GabMus** [comment]
bot, build org.gabmus.notorious

**Pull request closed without merging**